mobile-app/app/components/themed/ThemedScrollViewV2.tsx

```diff
@@ -0,0 +1,25 @@
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+import { tailwind } from '@tailwind'
+import { forwardRef } from 'react'
+
+import { ScrollView } from 'react-native'
+import { ThemedProps } from './index'
+
+type ThemedScrollViewProps = ScrollView['props'] & ThemedProps
+
+export const ThemedScrollViewV2 = forwardRef(function (props: ThemedScrollViewProps, ref: React.Ref<any>): JSX.Element {
```

Comment:
Why do we need a forwardref here?

Reply:
to allow scroll to top to work when tap on bottom tab menu